README.md

```diff
@@ -99,13 +111,18 @@ Note that in the following instructions, we assume your catkin workspace (in whi
    mkdir -p ~/catkin_ws/src
    ```
 
-1. Install mavros version 0.29.0 or above. Instructions to install it from sources can be found here: https://dev.px4.io/en/ros/mavros_installation.html. If you want to install using apt, be sure to check that the version is 0.29.0 or greater.
-
+1. Install mavros version 0.29.0 or above. Instructions to install it from sources can be found here: https://dev.px4.io/en/ros/mavros_installation.html.
+   If you want to install using apt, be sure to check that the version is 0.29.0 or greater.
+   * Melodic
+   ```bash
+   sudo apt install ros-melodic-mavros ros-melodic-mavros-extras
+   ```
+   * Kinetic
    ```bash
    sudo apt install ros-kinetic-mavros ros-kinetic-mavros-extras
    ```
 
-1. Install the geographiclib dataset
+1. Install the *geographiclib* dataset
```

Comment:
I presume this is needed for melodic too?

Reply:
yes